components/embl-logo/assets/logo-medium.svg

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I'll look at some SVGO gulp plugin to avoid:

a) the need to optimise this by hand
b) further PRs with all that Adobe cruft in the SVG